build.gradle

```diff
@@ -77,7 +77,3 @@ bintray {
         }
     }
 }
-
-wrapper {
```

Comment:
not actually needed, since now it ships with wrapper and wrapper config has its version